src/components/DatasetDownloads/index.jsx

```diff
@@ -1,16 +1,41 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import './datasetdownloads.scss';
+
+const DatasetDownloads = ({ dataDictionaryURL, dataDictionaryType, distributions }) => {
+  function trimDataDictionaryType() {
+    let splitDataDictionaryType = dataDictionaryType.split('/');
+    let type = splitDataDictionaryType.length > 1 ? splitDataDictionaryType[1] : splitDataDictionaryType[0];
+    return type.toUpperCase()
+  }
 
-const DatasetDownloads = ({ downloadURL, type }) => {
   return (
-    <div className="ds-u-margin-bottom--3 ds-u-padding--2 ds-u-border ds-u-border--1">
+    <div className="ds-u-margin-bottom--3 ds-u-padding--2 ds-u-border ds-u-border--1 dc-c-dataset-downloads">
```

Comment:
It looks like the design system has a utility class for this that we could put on the <ul> element:
`ds-c-list .ds-c-list--bare`

Instead of having a custom CSS file